pkg/simple/client/elasticsearch/esclient.go

```diff
@@ -501,28 +495,10 @@ func parseQueryResult(operation int, param QueryParameters, body []byte, query [
 		queryResult.Read = &readResult
 
 	case OperationStatistics:
-		var statisticsResult StatisticsResult
-		statisticsResult.Total = response.Hits.Total
-
-		var namespaceAggregations NamespaceAggregations
-		jsonIter.Unmarshal(response.Aggregations, &namespaceAggregations)
-
-		for _, namespace := range namespaceAggregations.NamespaceAggregation.Namespaces {
-			var namespaceResult NamespaceResult
-			namespaceResult.Namespace = namespace.Namespace
-			namespaceResult.Count = namespace.Count
-
-			for _, container := range namespace.ContainerAggregation.Containers {
-				var containerResult ContainerResult
-				containerResult.Container = container.Container
-				containerResult.Count = container.Count
-				namespaceResult.Containers = append(namespaceResult.Containers, containerResult)
-			}
-
-			statisticsResult.Namespaces = append(statisticsResult.Namespaces, namespaceResult)
-		}
-
-		queryResult.Statistics = &statisticsResult
+		var statisticsResponse StatisticsResponseAggregations
+		jsonIter.Unmarshal(response.Aggregations, &statisticsResponse)
```

Comment:
what if unmarshal failed , content in response can be anything

Reply:
thx, see the updated commit